### PR TITLE
Cut ropes keybind

### PR DESCRIPTION
**Pull request by BaerMitUmlaut**
**When merged this pull request will:**
- Add a keybind (default unbound) to cut fastroping ropes

### CODE REVIEW

File: addons/fastroping/XEH_postInit.sqf [hunk 1]
```diff
@@ -7,3 +7,12 @@
 [QGVAR(startFastRope), {
     [FUNC(fastRopeServerPFH), 0, _this] call CBA_fnc_addPerFrameHandler;
 }] call CBA_fnc_addEventHandler;
+
+["ACE3 Vehicles", QGVAR(cutRopes), localize LSTRING(Interaction_cutRopes), {
+    if ([vehicle ACE_player] call FUNC(canCutRopes)) then {
+        [vehicle ACE_player] call FUNC(cutRopes);
+        true
+    } else {
+        false
```
Comment (jonpas) [inline addons/fastroping/XEH_postInit.sqf]:
```
if (bla) exitWith {
    //cut
    true
};
false
```

Comment (BaerMitUmlaut) [inline addons/fastroping/XEH_postInit.sqf]: I don't see how that makes a difference.

Comment (jonpas) [inline addons/fastroping/XEH_postInit.sqf]: Ugh, I was thinking of something and don't know what, doing too many things at once. :anguished:

**Comment by jonpas** [comment]
PRed to proper branch in #4115

**Pull request closed without merging**